scanpy/tests/conftest.py

```diff
@@ -5,31 +5,18 @@
 from pathlib import Path
 from typing import TYPE_CHECKING
 
-import matplotlib as mpl
-
-mpl.use('agg')
-from matplotlib import pyplot
-from matplotlib.testing.compare import compare_images, make_test_filename
```

Comment:
Why import these inside of functions?

Reply:
Because no global state changes in modules.